src/MongoDB.Analyzer.Helpers/Builders/MqlGenerator.cs

```diff
@@ -23,8 +24,9 @@ public static class MqlGenerator
     {
 #pragma warning disable CS0169 // These fields are never used, they are needed to ensure that the relevant usings are not accidently removed
 #pragma warning disable IDE0051
-        private static readonly BsonType s_dummyRef1;
-        private static readonly TimeSpanUnits s_dummyRef2;
+        private static readonly Int32 s_dummyRef1; // using System
```

Comment:
Should we use C# `int` alias instead?

Reply:
Those dummy variable are used to capture the usings, so they are not removed by mistake.
`Int32 `is used for `System`, while `int `does not require `System`.